python/langsmith/client.py

```diff
@@ -491,6 +504,20 @@ def __init__(
         # Create a session and register a finalizer to close it
         session_ = session if session else requests.Session()
         self.session = session_
+        self.compress_traces = ls_utils.get_env_var("USE_RUN_COMPRESSION")
```

Comment:
Do we need this as well? Seems redundant with there being a buffer defined (if that's what we're checking forf)

Reply:
fixed